src/openforms/services/api/views.py

```diff
@@ -0,0 +1,27 @@
+from rest_framework import authentication, permissions, serializers, views
+from zgw_consumers.models import Service
+
+from openforms.api.views import ListMixin
+
+
+class ServiceSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Service
+        fields = (
+            "label",
+            "api_root",
+            "api_type",
+        )
+
+
+class ServiceListView(views.APIView, ListMixin):
```

Comment:
if possible, we prefer using viewsets:

```py
from rest_framework.viewsets import GenericViewSet
from rest_framework import mixins

class ServiceViewSet(mixins.ListModelMixin, GenericViewSet):
    ...
```
docs: https://www.django-rest-framework.org/api-guide/viewsets/

note that mixins should always appear left of the base classes due to how Python's MRO works. Please also stick to the patterns in other apps: a `api/viewsets.py` file and a `api/serializers.py` file + document the view/viewset using `drf_spectacular.utils.extend_schema(_view)`